src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -347,7 +348,7 @@ export class MatrixRTCSession extends TypedEventEmitter<MatrixRTCSessionEvent, M
     public async leaveRoomSession(timeout: number | undefined = undefined): Promise<boolean> {
         if (!this.isJoined()) {
             logger.info(`Not joined to session in room ${this.room.roomId}: ignoring leave call`);
-            return new Promise((resolve) => resolve(false));
+            return false;
```

Comment:
Please can you add JS/TS doc to the function def to actually say what the return value means?

Reply:
Good idea, done in 0383056de